src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -526,6 +543,48 @@ abstract class DatabaseTest {
     return row.id!!.also { inserted.projectIds.add(it) }
   }
 
+  private var nextDeliverableNumber: Int = 1
+
+  protected fun insertDeliverable(
+      createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+      deliverableCategoryId: DeliverableCategory = DeliverableCategory.FinancialViability,
+      deliverableTypeId: DeliverableType = DeliverableType.Document,
+      descriptionHtml: String? = "",
+      id: DeliverableId = nextDeliverableNumber.toIdWrapper { DeliverableId(it) },
+      isSensitive: Boolean = false,
+      isRequired: Boolean = false,
+      moduleId: ModuleId = inserted.moduleId,
+      name: String = "Deliverable $nextDeliverableNumber",
+      position: Int = 0,
+      subtitle: String? = "",
```

Comment:
This could default to null. (And we should probably remove subtitle from the data model anyway since it got removed from the requirements, but that's not a topic for this PR.)

Reply:
Will remove in a subsequent PR